bundles.md

```diff
@@ -285,3 +286,47 @@ New list of bitstreams:
   ]
 }
 ```
+
+## Primary bitstream
+
+The primary bitstream is exposed as a link, and can be modified
+
+**/api/core/bundles/<:uuid>/primaryBitstream**
+
+### Retrieve Primary bitstream
+
+The Primary bitstream can be retrieved using a GET, this may result in `null`
+
+**GET /api/core/bundles/<:uuid>/primaryBitstream**
+
+### Create Primary bitstream
+
+If the Primary bitstream doesn't exist, it can be created using a POST
+
+**POST /api/core/bundles/<:uuid>/primaryBitstream**
```

Comment:
I think we can get away with *just* supporting `PUT` on this endpoint... that way users don't need to first determine whether the primaryBitstream exists or not. 

Per our REST Contract README, we only claim to support `POST` "for collection associations (i.e. associations allowing for multiple resources". See https://github.com/DSpace/RestContract/blob/main/README.md#on-sub-path-of-a-single-resource-endpoint-associations

Since the "primaryBitstream" association of a Bundle only allows for *one* primary bitstream (not multiple) that means we don't need to support POST.  So, I think we can drop `POST` support and just use `PUT` if it's easier to do things that way.

Reply:
Since we're creating an association between a bundle and a bitstream, `PUT` doesn't seem appropriate to me to create the initial association. Since we're creating a new association, and not updating an existing one, I do believe we should stick with `POST`
Similarly, `DELETE` is used to delete the association

Reply:
I agree with @benbosman on that. It would be useful to reject a POST if a logo is already there as this could indicate a conflict with someone else that has uploaded the logo since the previous request of the bundle state.
This is also aligned with the current implementation of all the "special groups" (workflow, default item read, etc.) associated with a collection that are also a very similar scenario